.github/workflows/release.yml

```
@@ -0,0 +1,34 @@
+name: Release Artifacts
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)